st2reactor/st2reactor/rules/datatransform.py

```diff
@@ -60,7 +60,7 @@ def _construct_system_context(data, context):
         """Identify the system context in the data."""
         # The following regex will look for all occurrences of "{{system.*}}",
         # "{{ system.* }}", "{{ system.*}}", and "{{system.* }}" in the data.
-        regex = '{{\s*' + SYSTEM_PREFIX + '.(.*?)\s*}}'
+        regex = '{{\s*' + SYSTEM_KV_PREFIX + '.(.*?)\s*}}'
```

Comment:
I don't understand regex. If I have to fix this code I will replace with the keyvalue lookup :)